cypress/support/fragments/inventory/materialType/materialTypesPane.js

```diff
@@ -0,0 +1,39 @@
+import { Button, Pane, TextField, HTML, including, MultiColumnListRow } from '../../../../../interactors';
+import SettingsMenu from '../../settingsMenu';
+import ModalDeleteMaterialType from './modalDeleteMaterialType';
+import InteractorsTools from '../../../utils/interactorsTools';
+
+const pane = Pane('Material types');
+
+const verifyMaterialTypeIsPresented = (materialTypeName) => {
+  cy.expect(pane.find(HTML(including(materialTypeName))).exists());
+};
+
+const verifyMaterialTypeIsDeleted = (newMaterialTypeName) => {
+  cy.expect(pane.find(HTML(including(newMaterialTypeName))).absent());
+};
+
+export default {
+  createNewMaterialType:(materialTypeName) => {
+    cy.visit(SettingsMenu.materialTypePath);
+    cy.do(Button('+ New').click());
+    cy.do(TextField({ placeholder: 'name' }).fillIn(materialTypeName));
+    cy.do(Button('Save').click());
+    verifyMaterialTypeIsPresented(materialTypeName);
```

Comment:
may be it's better to move verification into test

Reply:
Done